version.go

```diff
@@ -35,6 +35,16 @@ func (version *ArtifactVersion) refresh() {
 	}
 }
 
+// IsStableVersion returns true if the version indicates an official release.
+func (version *ArtifactVersion) IsDevelopment() (bool, error) {
```

Comment:
I would prefer not to use the word development here because it can be ambiguous, especially across multiple versioning schemes - does it mean development build, development release, development series, etc.

Reply:
Yeah that makes sense, I can't really decide what to name it though. It's just if the minor number doesn't indicate a release. Do you have any suggestions?

Reply:
I can make this private also